README.md

```diff
@@ -1,65 +1,95 @@
 # Desktop Embedding for Flutter
 
 This project was originally created to develop Windows, macOS, and Linux
-implementations of [Flutter](https://github.com/flutter/flutter). That work has
-since become part of the
-[Flutter engine repository](https://github.com/flutter/engine), and this
+embeddings of [Flutter](https://github.com/flutter/flutter). That work has
+since [become part of
+Flutter](https://github.com/flutter/flutter/wiki/Desktop-shells), and this
 project is now just an example of, and test environment for, building
-applications using those libraries.
+applications using those libraries in their current state.
 
-For information about the shells themselves, see the [Flutter page about
-desktop support](https://github.com/flutter/flutter/wiki/Desktop-shells).
+As explained in the link above, desktop libraries are still in early stages.
+**The code here is not stable, nor intended for production use.**
 
-## How to Use This Code
+## Setting Up
 
-_If you have an existing Flutter app and just want to get it running, see
-the [quick start](Quick-Start.md) page before continuing._
+Everything in this project requires [enabling desktop support in
```

Comment:
It's confusing to me to see a link to this same info on another doc that says essentially the same thing. I'm not sure why I'm being redirected to another page in this case.

Reply:
I link to the other page to provide context for why these steps are necessary.

This version includes more step-by-step details, such as the exact commands to set the environment variable.